osd.c

```diff
@@ -108,9 +108,9 @@ void modeset_paint_buffer(struct modeset_buf *buf) {
 			}
 			avg_bw = avg_bw / avg_cnt;
 			if (avg_bw < 1000) {
-				sprintf(msg, "%.2f Kbps", avg_bw / 125 );
+				sprintf(msg, "%.2f Kbps", avg_bw / 75 );
```

Comment:
That's  incorrect, the average bandwidth is bytes/second so 125 or 125000 is the right value.
Where is 75 is coming from?

avg_bw is populated from frame->size() which is an array of uin8_t (= a byte).
https://github.com/gehee/FPVue_rk/blob/main/main.cpp#L373